docs/developers/schema_versions.md

```diff
@@ -24,3 +26,15 @@ To aid with this process, the following guidelines are provided:
 In general, a schema version should be adjusted if and only if there's a change in the schema's definition. This is typically reserved for major releases _(e.g. upgrading from iRODS 4.2 to iRODS 4.3)_.
 
 A _change in the schema definition_ is defined as the addition, modification, and/or removal of a JSON property or database table/column.
+
+### iRODS 5 and later
+
+!!! Important
+    Starting with iRODS 5, version numbers consist of three segments, _X.Y.Z_, where _X_ is the major version number, _Y_ is the minor version number, and _Z_ is the patch version number. A major version change consists of the major version number being incremented. For example, upgrading from iRODS 4.3.4 to iRODS 5.0.0 or iRODS 5.1.0 to iRODS 6.0.0.
+
+iRODS 5 consolidates management of the schema versions to the top-level CMakeLists.txt file, making it easy to adjust version numbers. There are two CMake variables, each serving different purposes. They are as follows:
+
+- `IRODS_CATALOG_SCHEMA_VERSION`: This option controls the schema version of the catalog
+- `IRODS_CONFIGURATION_SCHEMA_VERSION`: This option controls the schema version of the configuration files
+
+While iRODS 5 improves management of version numbers, the guidelines defined for iRODS 4 still apply.
```

Comment:
Maybe we move this line up closer to the notice about how versioning works in iRODS 5? Does that seem weird?

Reply:
top might be clearer.

Reply:
Updated.